stacks/services/APIGateway.ts

```diff
@@ -26,6 +28,16 @@ export const createSiriApi = (stack: Stack, siriSXBucket: Bucket, hostedZone: IH
                 description: "API to retrieve Siri SX XML data",
```

Comment:
We should probably update this description to include stats about SIRI data

Reply:
done